### PR TITLE
[SC-210100] GISAID Terms of use updates.

### PR DESCRIPTION
**Pull request by ehoops-cz**
### Summary:
- **What:** `Update to tree creation modal to acknowledge gisaid.`
- **Ticket:** [sc210100](https://app.shortcut.com/genepi/story/210100)
- **Env:** `<rdev link>`

### Demos:
Screen Shot 2022-10-07 at 3 30 47 PM

### Notes:
Includes new copy
Includes update to use scroll="body"
Includes link on Gisaid logo

### Checklist:
- [x] I merged latest `<base branch>`
- [x] I manually verified the change
- [x] I added labels to my PR
- [x] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/frontend/src/common/styles/basicStyle.ts [hunk 1]
```diff
@@ -82,5 +82,8 @@ export const transparentScrollbars = (): string => {
     &::-webkit-scrollbar-track {
       background: transparent;
     }
+    &::-webkit-scrollbar {
+      display: none;
+    }
```
Comment (ehoops-cz) [inline src/frontend/src/common/styles/basicStyle.ts:87]: @mmmmmmaya I'm not sure why I needed this - the transparent background wasn't working.  Are there any red flags with adding this?  It definitely still scrolls.

**Comment by ehoops-cz** [comment]
@Janeece Hiding the scrollbars doesn't work in Firefox (they don't let us).  This was the case before, but the content scroll doesn't look too bad. The scroll="body" looks pretty bad.

Before: 
Screen Shot 2022-10-07 at 3 41 45 PM

After:
Screen Shot 2022-10-07 at 3 42 19 PM

Most of our users are not on Firefox, so I'm not sure if this is something we should worry about.  Thoughts?

**Comment by ehoops-cz** [comment]
Approved by Christine and Janeece over slack

File: src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/index.tsx [hunk 2]
```diff
@@ -303,11 +306,28 @@ export const CreateNSTreeModal = ({
           <Attribution>
             Built in partnership with <NextstrainLogo />, enabled by data
             from&nbsp;
-            <ImageSizer>
-              <Image src={GisaidLogo} />
-            </ImageSizer>
+            <Link href="https://gisaid.org/" target="_blank">
```
Comment (mmmmmmaya) [inline src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/index.tsx]: `NewTabLink`?

File: src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/style.ts [hunk 3]
```diff
@@ -16,23 +16,50 @@ import DialogTitle from "src/common/components/library/Dialog/components/DialogT
...
+  ${gray500}
+`;
+
+export const SpacedAcknowledgements = styled(Acknowledgements)`
+  ${(props) => {
+    const spaces = getSpaces(props);
+
+    return `
+    margin: 0 0 ${spaces?.s}px;
+  `;
```
Comment (mmmmmmaya) [inline src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/style.ts]:
```suggestion
      margin: 0 0 ${spaces?.s}px;
    `;
```

**Review by mmmmmmaya — APPROVED** [review]

**Pull request merged**